docs/data-management.rst

```diff
@@ -207,41 +312,80 @@ If you are an owner of the data, you can move the data between the groups you ar
+#. Follow further the steps described in the section
+   ``Move data between groups: owners of data`` above, taking note of the
+   ``Not included`` objects.
 
-#. When creating new Datasets or Projects during the move, note that these containers will belong to the owner of the data, not yourself. Also the links between the new containers and the moved data will belong to the owner of the data. This should help to facilitate a smooth workflow, retaining the data handling possibilities such as reorganizing the data, renaming the containers you created for them etc. for the owner of the data. 
+#. When creating new Datasets or Projects during the move,
+   note that these containers will belong to the owner of the data, 
+   not yourself. Also the links between the new containers
+   and the moved data will belong to the owner of the data. 
+   This should help to facilitate a smooth workflow,
+   retaining data-handling possibilities such as reorganizing the data,
+   renaming the containers you created for them etc. for the owner of the data. 
 
 .. _Duplicatecli:
 
 *Command Line: Duplicating objects*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can duplicate objects in OMERO. The functionality is available only on the CLI for now. The duplication creates a full copy of the objects as if they were created independently. When desired, the newly duplicated objects can additionally be linked to other objects. When the duplicate is later deleted, it will not have an influence on the original, which stays preserved. Similarly, when the original is later deleted, the duplicate stays preserved as well.
-
-In case of Image objects, which have image files linked, the duplication creates a new image file which is linked to the original image file by a hard link. This means every duplication of an Image increases the number of hard links on the image file in Managed Repository of OMERO, but does not duplicate the image file itself, and thus does not increase the storage demands too much. 
-
-In case of File Attachment objects though, which also have files to them, each duplication will duplicate the linked file, thus doubling the storage space necessary for these File Attachment files.
+You can duplicate objects in OMERO. The functionality is available only
+on the CLI for now. The duplication creates a full copy of the objects
+as if they were created independently. 
+When desired, the newly duplicated objects can additionally be linked
+to other objects. When the duplicate is later deleted, it will not have
+an influence on the original, which stays preserved.
+Similarly, when the original is later deleted,
+the duplicate stays preserved as well.
+
+In case of Image objects, which have image files linked,
+the duplication creates a new image file which is linked
+to the original image file by a hard link. This means every
```

Comment:
What happens if the managed repository is split across two disks (e.g. because it was expanded)?

Reply:
> What happens if the managed repository is split across two disks (e.g. because it was expanded)?

Attempted linking will fail, but the copy will proceed, creating a new original file, right @mtbc ?
Maybe I should insert that the linking behaviour is just an attempt.

Reply:
Yes, that is what *should* happen at least!

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/9458b5f270c3f5ed0e416baa5db828cbf8620b9a - @mtbc could you please have a look at the formulation in this commit ? Not sure about how "rare" such cases are (see text).

Reply:
Looks good but does rather challenge the "This means ..." following.

Likelihood has everything to do with the history of how the institution has expanded its repository volumes over the years and split users or eras across it. Generally I'd expect hard linking to be possible if the data's still in its usual group and not ancient.

Reply:
> but does rather challenge the "This means ..." following.

@mtbc 

Fixed in https://github.com/ome/omero-guide-introduction/pull/17/commits/3166a9349ee79f30482fd278b2e72b203591649c